docs/source/authentication.rst

```diff
@@ -134,18 +134,9 @@ The following is a list of auth backends for the community edition to help get t
 * `Flat File <https://github.com/StackStorm/st2-auth-backend-flat-file>`_
 * `OpenStack Keystone <https://github.com/StackStorm/st2-auth-backend-keystone>`_
 
-LDAP (Enterprise Edition)
--------------------------
-.. note::
-
-  Prior to StackStorm 3.3, Extreme Networks provided a commercial version(Extreme Workflow Composer) of the StackStorm automation platform including an LDAP auth backend.
-
-Extreme Workflow Composer was donated to Linux Foundation, but those components are not yet integrated into the StackStorm release. Please visit https://stackstorm.com/2020/05/27/extreme-networks-donates-ewc-to-linux-foundation/ for more details.
-
-  These components are NOT available in the StackStorm 3.3 release, but will be integrated into the community edition of StackStorm in future release(s).
-
 LDAP
-^^^^
+----
+
 The LDAP backend authenticates the user against an LDAP server. The following is a list of
```

Comment:
Let's reference the https://github.com/stackstorm/st2-auth-ldap repository here as a source of truth for the LDAP auth backend as it's now open.

Following that,
@amanda11 @StackStorm/maintainers do you think it's a good idea to also include more examples here from the https://github.com/stackstorm/st2-auth-ldap#configuration-examples repo for the users to find it easier?

Reply:
@armab One thing, is there is more configuration parameters listed on the st2-auth-ldap page than there are on this page for the LDAP section. I don't know which list is correct!

Do we link to a particular verison of st2-auth-ldap or just the latest. If we pick up a version that isn't the latest, then it may be best to add further information to the main stackstorm pages (as then if st2-auth-ldap package has changes that aren't in a StackStorm release then people won't get confused). But at the moment I think we are just pointing to the latest st2-auth-ldap - so in that case a link out would make sense.

Reply:
@amanda11 That's a good catch! So sounds like the current `st2docs` LDAP documentation was indeed heavily unsync with the https://github.com/stackstorm/st2-auth-ldap repository which is a source of truth. We use master/latest version.

@hnanchahal Based on that, please update the LDAP `st2docs` in this PR to reflect the actual https://github.com/stackstorm/st2-auth-ldap repository documentation and parameters.

Reply:
Thanks for updating the old LDAP docs! Looks much better now @hnanchahal !